src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -447,7 +447,7 @@ export default class AircraftCommander {
 
         // Set the runway to taxi to
         if (!taxiDestination) {
-            taxiDestination = this._airportController.airport_get().runway;
+            taxiDestination = this._airportController.airport_get().departureRunway.name;
```

Comment:
I would much rather see:
```javascript
const airport = this._airportController.airport_get();
taxiDestination = airport.departureRunway.name;
``` 
It makes the intent a little clearer and eventually we would just store the current airport in the constructor so we don't have to use `airport_get()` all the time.